src/user/views/displayImage.mustache

```diff
@@ -61,85 +72,74 @@
-                imageNameTag.innerText = imageName;
-                currentImage.setAttribute('src', imageSource);
-            }
-
-            function setPreviousImage(imageIndex) {
+        
+            function setPreviousImage(currentIndex) {
                 console.log(previousImage);
-                if (imageIndex < 0) {
+                if (currentIndex < 0) {
                     previousImage.style.visibility = 'hidden';
                     return;
                 }
                 previousImage.style.visibility = 'visible';
-                let imageSource = '/public' + files[imageIndex];
+                let imageSource = '/public' + files[currentIndex];
                 previousImage.setAttribute('src', imageSource);
                 console.log(previousImage);
             }
 
-            function setNextImage(imageIndex) {
-                if (imageIndex > files.length - 1) {
+            function setNextImage(currentIndex) {
+                if (currentIndex > files.length - 1) {
                     nextImage.style.visibility = 'hidden';
                     return;
                 }
                 nextImage.style.visibility = 'visible';
-                let imageSource = '/public' + files[imageIndex];
+                let imageSource = '/public' + files[currentIndex];
                 //console.log(imageSource);
                 nextImage.setAttribute('src', imageSource);
             }
 
-            function setImages(currentIndex) {
-                setCurrentImage(currentIndex);
-                setPreviousImage(currentIndex - 1);
-                setNextImage(currentIndex + 1);
-            }
-
-            function moveLeft() {
-                currentFileIndex = Math.max(0, currentFileIndex - 1);
-                setImages(currentFileIndex);
-            }
-
-            function moveRight() {
-                currentFileIndex = Math.min(files.length - 1, currentFileIndex + 1);
-                setImages(currentFileIndex);
+            function slideAfterEffects(currentIndex) {
+                const imageName = baseName(files[currentIndex]);
+                titleTag.innerText = imageName;
+                imageNameTag.innerText = imageName;
+                setPreviousImage(currentIndex-1);
+                setNextImage(currentIndex+1);
             }
 
     </script>
+
     <script>
-            function decreaseCurrentFileIndex(){
-                currentFileIndex = Math.max(0,currentFileIndex-1);
-                setImages(currentFileIndex);
-            }
-            function increaseCurrentFileIndex(){
-                currentFileIndex = Math.min(files.length-1,currentFileIndex+1);
-                setImages(currentFileIndex);
+        let galleryItems= [];
+        for(file of files)
+        {
+            let temp = {};
+            temp.title = baseName(file);
+            temp.href = '/public' + file;
+            galleryItems.push(temp);
+        }
+        var gallery = blueimp.Gallery(
```

Comment:
- Image name is already being shown in the title bar and having image name shown inside the image also feels kinda repetitive
- Same is the case with arrows as the image thumbnails look a lot better and we don't need arrows, the arrows also feels repetitive and unrequired

If there is a `blueimp` settings, then I would propose these changes to be incorporated before merging this branch to master